kinisi/analyzer.py

```diff
@@ -120,14 +120,13 @@ def _from_pymatgen(cls,
             return cls(u.delta_t, u.disp_3d, u._n_o, u.volume, **kwargs)
         else:
             raise ValueError('The dtype specified was not recognised, please consult the kinisi documentation.')
-        
+
     @classmethod
     def _from_ase(cls,
-                       trajectory: List[Union['ase.atoms.Atoms',
-                                              List['ase.atoms.Atoms']]],
-                       parser_params: dict,
-                       dtype: str = None,
-                       **kwargs):
+                  trajectory: List[Union['ase.atoms.Atoms', List['ase.atoms.Atoms']]],
```

Comment:
Once you have made some changes, can you run `yapf` on the code which will do the formatting. If I remember, the command is 
```
yapf -i kinisi/*
```